### ISSUE TITLE
Do a NULL modeset on connector unplug

### ISSUE THREAD

**Issue by emersion** [swaywm/wlroots#1706]
I've learned today that if userspace doesn't do a NULL modeset when a connector changes its state from connected to disconnected, the pipe on the kernel side will remain active and will continue to send pixels (to /dev/null, wasting energy). This is apparently a "feature" for userspace that doesn't support hotplug events (so that after unplugging-replugging the screen receives again an image).

tl;dr we need to do a NULL modeset.
